examples/hacker_news/hacker_news_tests/test_jobs/test_download_job.py

```diff
@@ -0,0 +1,33 @@
+import tempfile
+
+from dagster import ResourceDefinition, fs_io_manager
+from hacker_news.jobs.download_job import configured_pyspark, download_graph
+from hacker_news.resources.hn_resource import hn_snapshot_client
+from hacker_news.resources.parquet_io_manager import partitioned_parquet_io_manager
+
+
+def test_download():
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        result = download_graph.to_job(
```

Comment:
Both `to_job` and `execute_in_process` accept resource and config arguments.  For consistency, probably makes sense to supply both in either one or the other.  FWIW, I usually forgo `to_job` entirely and supply both in `execute_in_process` when testing a graph.

Reply:
for context on why i did it this way: `config` for `graph` cant target `resources` so I did it this way to avoid having to `configured` each resource individually

Reply:
@alangenfeld - so you're saying that `GraphDefinition.execute_in_process` doesn't accept resource config in its config argument?  I think I was imagining that the implementation of `GraphDefinition.execute_in_process` would look something like `self.to_job(run_config=config, resource_defs=resource_defs).execute_in_process()`.  Is there a reason we don't do it that way?

Reply:
https://github.com/dagster-io/dagster/pull/4646

I think the thinking was you are executing the graph directly so the config is scoped to just that. I think we can make it work the way you describe if we rename it to `run_config`

Reply:
Created an issue to have that discussion on: https://github.com/dagster-io/dagster/issues/4719.